native/src/main/java/io/ballerina/stdlib/grpc/logging/GrpcLogManager.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.grpc.logging;
+
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BString;
+import io.ballerina.stdlib.grpc.logging.formatters.GrpcAccessLogFormatter;
+import io.ballerina.stdlib.grpc.logging.formatters.GrpcTraceLogFormatter;
+import io.ballerina.stdlib.grpc.logging.formatters.JsonLogFormatter;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+import java.util.logging.SocketHandler;
+
+import static io.ballerina.stdlib.grpc.GrpcConstants.GRPC_LOG_CONSOLE;
+import static io.ballerina.stdlib.grpc.GrpcConstants.GRPC_LOG_FILE_PATH;
+import static io.ballerina.stdlib.grpc.GrpcConstants.GRPC_TRACE_LOG_HOST;
+import static io.ballerina.stdlib.grpc.GrpcConstants.GRPC_TRACE_LOG_PORT;
+import static io.ballerina.stdlib.grpc.GrpcConstants.HTTP_ACCESS_LOG;
+import static io.ballerina.stdlib.grpc.GrpcConstants.HTTP_ACCESS_LOG_ENABLED;
+import static io.ballerina.stdlib.grpc.GrpcConstants.HTTP_TRACE_LOG;
+import static io.ballerina.stdlib.grpc.GrpcConstants.HTTP_TRACE_LOG_ENABLED;
+
+/**
+ * Java util logging manager for ballerina which overrides the readConfiguration method to replace placeholders
+ * having system or environment variables.
+ */
+public class GrpcLogManager extends LogManager {
+
+    static {
+        // loads logging.properties from the classpath
+        try (InputStream is = GrpcLogManager.class.getClassLoader().
+                getResourceAsStream("logging.properties")) {
+            LogManager.getLogManager().readConfiguration(is);
+        } catch (IOException e) {
+            throw new RuntimeException("failed to read logging.properties file from the classpath", e);
```

Comment:
```suggestion
            throw new RuntimeException("Failed to read logging.properties file from the classpath", e);
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/413/commits/303299de9612deb38959f15b7389ed952825454a